### DISCUSSION TITLE
How do we Handlebar to work with with Magento 2.0 eCommerce themes

### DISCUSSION THREAD

**Discussion by billabongboy69** [handlebars-lang/handlebars.js#1881] [Q&A]
Forgive me if this is not the right place.

We are trying to inject and render widgets on Magento 2.0. Some of our widgets use Handlebar, but it doesn't seem to be working. We've asked the Magento community and no-one has an answer to date, so thought I'd ask here to see if anyone might be able to provide some guidance.

In the words of our developer:
How can we load handlebars Js library in Magento 2?

I have tried

- Require JS
- XML
- Direct Script in html file. But none of these methods worked. 

Have error in console Handlebars is not defined....

Any guidance or suggestions would be appreciated.

**Comment by jaylinski** [comment]
Direct script should work fine. Just be sure to wait until the Handlebars script is loaded before trying to use it.

**Reply by katnavipasha** [reply]
@jaylinski 
Tried document ready to wait for library to load before using handlebar. But still same error handlebar not defined.

**Reply by billabongboy69** [reply]
@katnavipasha any screenshot you can shar or additional information for Handlebar discussion here?
